master/pom.xml

```diff
@@ -1789,6 +1789,16 @@
 
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-eclipse-plugin</artifactId>
+        <version>2.10</version>
+        <!-- https://issues.apache.org/jira/browse/MECLIPSE-726 -->
+        <configuration>
+          <useProjectReferences>true</useProjectReferences>
```

Comment:
I think this is the default anyway: https://maven.apache.org/plugins/maven-eclipse-plugin/eclipse-mojo.html#useProjectReferences